docs/internal/Assignment-Process.md

```diff
@@ -27,14 +27,12 @@ Review assignment can have the following statuses
 
 To also determine if this assignment can be Self-assigned, 3 other fields come on the scene:
 
-- `is_self_assignable` -> Can be self assigned (if not locked - how we prevent other self-assigners after one is already assigned)
-- `is_locked` -> Can be used to either prevent self-assignment or to generate a Review with status = LOCKED & assignment status is ASSIGNED\*
+- `is_self_assignable` -> Can be self assigned (if there is `available_sections` to be selected - how we prevent other self-assigning for sections already assigned)
+- `is_locked` -> Used to prevent a Review from being submitted - in case another reviewer in same stage already send a response to Applicant (decision to "Send back to applicant")
```

Comment:
Updated references to how we check if `is_self_assignable` and can self assign
Also updated reference to `is_locked` now only used for checking if Review is submitable